mindjunkies/courses/views.py

```
@@ -15,6 +15,8 @@
 from .models import Course, Enrollment
 from .forms import CourseForm, RatingForm
 from .models import CourseToken, LastVisitedCourse, Rating
+from django.dispatch import Signal
```

Ruff (0.8.2)

`django.dispatch.Signal` imported but unused

Remove unused import: `django.dispatch.Signal`

(F401)